Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -1258,6 +1255,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MODULEMAP_FILE = "$(SRCROOT)/../Sources/Datadog/Datadog.modulemap";
 				PRODUCT_BUNDLE_IDENTIFIER = com.datadogqh.Datadog;
```

Comment:
I think now we don't need this:
```
SWIFT_INCLUDE_PATHS = "$(SRCROOT)/DatadogPrivate";
```
Which is "Import Paths" Build Setting:

<img width="1150" alt="Screenshot 2020-04-17 at 17 12 36" src="https://user-images.githubusercontent.com/2358722/79584448-b1090c80-80ce-11ea-9be1-23c69fc25867.png">

Reply:
you are right 👍 
removing it